pages/pipelines/standard_and_enterprise_features.md.erb

```diff
@@ -0,0 +1,25 @@
+# Standard and Enterprise Features  
+
+**Buildkite Free Plan** is free for open-source projects, students, and teaching organizations of any size and provides:  
+
+* Unlimited agents
+* Unlimited builds
+* Unlimited artifacts
+
+**Buildkite Standard Plan** (up to 200 users per organization) adds:  
+
+* [Teams permissions](/docs/pipelines/permissions#permissions-with-teams)
+* [Single Sign On](/docs/integrations/sso)
+* Priority email support
+
+**Buildkite Enterprise Plan** (100 users per organization with an expansion option) adds extended administration options: 
+
+* [Audit logging](/docs/pipelines/audit-log#main)
+* [Member permissions management tool](/docs/pipelines/permissions#member-permissions)
+* Extended logs storage options
+* Live chat support
+* Extended legal and accounting options
+ 
+Buildkite Enterprise license covers every user of your code repository provider organization (for example, GitHub organization) or anyone who directly or indirectly interacts with your Buildkite organization. This includes users who log in to your Buildkite organization and external users who trigger builds via merging pull requests. Suppose a Buildkite Enterprise organization exceeds 100 users. In that case, new members can still be invited (or — if SSO is enabled — new members will still be auto-created upon signing in) for extra monthly charges.
```

Comment:
Do we want to spell this out in plain text like this? I can think of circumstances where it might not be an ideal fit.

```suggestion
The Buildkite Enterprise license covers every user of your code repository provider organization and anyone who directly or indirectly interacts with your Buildkite organization. This includes users who log in to your Buildkite organization, and external users who trigger builds via pull requests. If your Buildkite Enterprise organization exceeds 100 users, new members can still be invited and will be billed pro-rata.
```